org.kuchelmeister.toolbxtuner.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id" : "org.kuchelmeister.toolbxtuner",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "toolbx-tuner",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.Flatpak"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
+        ],
```

Comment:
this is not allowed. You need to use flatpak-builder-tools to generate a manifest for the cargo dependencies if they are not vendored.

Reply:
Is there any recommendations for an ideal way to do that? Using flatpak-builder-tools results in a rather large manifest.

Reply:
This is the recommend way. You can include other files (like the one generated from flatpak-builder-tools) in your main file.

Reply:
1. said manifest being autogenerated that's not a problem (it's meant to just be included, and only updated with the same tools)
2. you can reduce your dependencies
3. you can use a tarball with vendored crates, which I think the default meson templates do.